doc/cygnus-twitter/user_and_programmer_guide/user_guide.md

```diff
@@ -0,0 +1,157 @@
+#<a name="top"></a>Introduction
```

Comment:
I would move this file to `doc/cygnus-twitter/installation_and_administration_guide`, having at least the following functions:

* Installation
* Configuration
* Running
* Testing

Then other sections may appear. Finally, sanity checks and diagnosis procedures are welcome as well.

You may use as a reference: https://github.com/telefonicaid/fiware-cygnus/tree/develop/doc/cygnus-ngsi/installation_and_administration_guide